pkgs/by-name/li/livekit-cli/package.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "livekit-cli";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "livekit";
+    repo = "livekit-cli";
+    rev = "v${version}";
+    hash = "sha256-zHnygEiT7IGHoJ5Mz2fFbIbccZV4XAx608e+VH/avEc=";
+  };
+
+  vendorHash = "sha256-KgUVDHarVGaUqOnKe2Tbko0CmLqlC322a34HA9cLqz4=";
+
+  subPackages = [ "cmd/livekit-cli" ];
+
+  meta = with lib; {
+    description = "Command line interface to LiveKit";
+    homepage = "https://livekit.io/";
+    mainProgram = "livekit-cli";
+    license = with licenses; [ asl20 ];
```

I suggest following the recommended format for licenses.
```suggestion
    license = licenses.asl20 ;
```

---

Thanks for the suggestion, applied and squashed into a single commit. Could you please point me to the documentation where that recommended format is described?

---

see https://nixos.org/manual/nixpkgs/stable/#var-meta-license
> If the license field is in the form of a list representation, then it means that parts of the package are licensed differently.

---

This is not a requirement, tho. It merely says something clear: a list with multiple elements implies multiple individuals.